s2and/model.py

```diff
@@ -219,13 +220,16 @@ def make_distance_matrices(
         logger.info("Initializing pairwise_probas")
         # initialize pairwise_probas with correctly size arrays
         pairwise_probas = {}
+        num_pairs = 0
         for block_key, signatures in block_dict.items():
             block_size = len(signatures)
             if isinstance(self.cluster_model, FastCluster):
                 # flattened pdist style
                 pairwise_proba = np.zeros(int(block_size * (block_size - 1) / 2), dtype=np.float16)
+                num_pairs += int(block_size * (block_size - 1) / 2
             else:
                 pairwise_proba = np.zeros((block_size, block_size), dtype=np.float16)
+                num_pairs += block_size * block_size
```

Comment:
Is this the right number in this case?

Reply:
No, `num_pairs` is actually the same in both instances. This just takes care of the shape. If you look here you can see that we add the transpose, which means only the upper triangular gets computed: https://github.com/allenai/S2AND/blob/298a5ed9cfee0d7375f36c640ef8d1d08aaade1d/s2and/model.py#L329